Vagrantfile

```diff
@@ -0,0 +1,15 @@
+# -*- mode: ruby -*-
```

Comment:
I recommend naming this file ```Vagrantfile.example``` instead, and in the readme telling devs to copy it to Vagrantfile (and adding Vagrantfile to the ```.gitignore```.) This way, devs are free to customize their Vagrantfile, but they also have a really nice easy way to get started.